src/test/java/com/shinesolutions/aemorchestrator/service/AemInstanceHelperServiceTest.java

```diff
@@ -492,19 +486,19 @@ public void testGetPublishIdForPairedDispatcherWithFoundPair() throws Exception
         when(awsHelperService.getTags(instance1)).thenReturn(tagsWithoutPair);
         when(awsHelperService.getTags(instance2)).thenReturn(tagsMissingPair); 
         when(awsHelperService.getTags(instance3)).thenReturn(tagsWithPair); //Instance 3 is the winner
-        when(awsHelperService.getTags(instance4)).thenReturn(tagsWithoutPair);
-        
+
         String resultInstanceId = aemHelperService.getPublishIdForPairedDispatcher(dispatcherId);
-        
+
+        verify(awsHelperService, never()).getTags(instance4);
```

Comment:
Since `instance3` is the winner, `getTags(instance4)` should never be called. Can you verify if my thinking is correct here @cliffano . Thank you. :)